File: js/test_money.js
const assert = require('assert');

class Money {
    constructor(amount, currency) {
        this.amount = amount;
        this.currency = currency;
    }

    times(multiplier) {
        return new Money(this.amount * multiplier,
                        this.currency);
    }

    divide(divisor) {
        return new Money(this.amount / divisor,
                        this.currency);
    }
}

let originalMoney = new Money(4002, "KRW")
let expectedMoneyAfterDivision = new Money(1000.5, "KRW")
assert.deepStrictEqual(originalMoney.divide(4), expectedMoneyAfterDivision)

File: py/test_money.py
import unittest

from money import Money
from portfolio import Portfolio

class TestMoney(unittest.TestCase):
    def testMultiplicationInDollars(self):
        fiver = Money(5, "USD")
        tenner = Money(10, "USD")
        self.assertEqual(tenner, fiver.times(2))

    def testDivision(self):
        originalMoney = Money(4002, "KRW")
        expectedMoneyAfterDivision = Money(1000.5, "KRW")
        self.assertEqual(expectedMoneyAfterDivision,
                        originalMoney.divide(4))

if __name__ == '__main__':
    unittest.main()